.github/workflows/ci-pr.yaml

```diff
@@ -15,6 +15,7 @@
 name: "Continuous Integration - Pull Request"
 on:
   pull_request:
+    types: [review_requested, ready_for_review]
```

Comment:
That definitely covers the initial creation of the PR (or when a draft PR gets converted to a "ready to review" PR) but does it also cover the cases where the PR is rebased, gets new commits, etc? (in other words, will those actions also run this Action?)

Reply:
yes those actions will run